{
  "table": 0,
  "error": "The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().",
  "trace": "Traceback (most recent call last):\n  File \"/Users/macbookpro/Documents/bankbot/banklytik/statements/direct_processor.py\", line 131, in process_tables_directly\n    cleaned = _clean_table_dataframe(df_table, normalize_text, parse_date_str, clean_amount, extract_channel)\n  File \"/Users/macbookpro/Documents/bankbot/banklytik/statements/direct_processor.py\", line 221, in _clean_table_dataframe\n    df[\"balance\"] = df.get(\"balance\", pd.Series([\"\"] * len(df))).apply(clean_amount)\n  File \"/Users/macbookpro/Documents/anaconda3/envs/bank/lib/python3.10/site-packages/pandas/core/frame.py\", line 10374, in apply\n    return op.apply().__finalize__(self, method=\"apply\")\n  File \"/Users/macbookpro/Documents/anaconda3/envs/bank/lib/python3.10/site-packages/pandas/core/apply.py\", line 916, in apply\n    return self.apply_standard()\n  File \"/Users/macbookpro/Documents/anaconda3/envs/bank/lib/python3.10/site-packages/pandas/core/apply.py\", line 1063, in apply_standard\n    results, res_index = self.apply_series_generator()\n  File \"/Users/macbookpro/Documents/anaconda3/envs/bank/lib/python3.10/site-packages/pandas/core/apply.py\", line 1081, in apply_series_generator\n    results[i] = self.func(v, *self.args, **self.kwargs)\n  File \"/Users/macbookpro/Documents/bankbot/banklytik/statements/cleaning_utils.py\", line 165, in clean_amount\n    if pd.isna(value):\n  File \"/Users/macbookpro/Documents/anaconda3/envs/bank/lib/python3.10/site-packages/pandas/core/generic.py\", line 1577, in __nonzero__\n    raise ValueError(\nValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().\n"
}